en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
-        
-        Click **Save & Done**.
-        
-        Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+            
+    6. Click **Save & Done**.
     
-7. To cast all the data retrieved via the API connected to the service before further processing, add a new custom statement as follows:
+       Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+
+7. Add a new custom statement as follows to cast all the data retrieved via the API connected to the service before further processing:
 
     1. Click the last **+** icon in your low-code diagram.
 
         ![Add New Statement](../assets/img/services/add-custom-statement.png)
     
-    2. Click **Other** and then enter the following in the **Statement** field.
-    
-        ```
+    2. Click **Other** and then enter the following in the **Statement** field:
+
+        ```ballerina
         json[] jsonArray = <json[]>jsonPayload;
         json[] response = [];
         ```
-        Then click **Save**.
-    
-8. To filter the dates on which the active cases have exceeded 5,000, let's add a statement of the `ForEach` type as follows: 
+    3. Click **Save**.
+
+8. Add a ForEach statement as follows to filter the dates where the active cases have exceeded 5,000:
 
     1. Click the last **+** icon in your low-code diagram.
     
     2. Click **ForEach**.
 
     3. In the **Iterable Expression** field, enter `jsonArray`. Then click **Save**.
     
-    4. To apply the filter mentioned, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
+    4. To apply the filter, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
 
         ![Add Custom Statement After Foreach Statement](../assets/img/services/add-custom-statement-after-foreach-statement.png)
     
-    5. In the form that appears, click **Other**. Then enter the following in the **Statement** field.
+    5. In the form that appears, click **Other**. Then enter the following in the **Statement** field:
 
         ```ballerina
         int active = <int>(check item.Active);
         if (active > 5000) {
             response.push(item);
         }
         ```
-        
-        Then click **Save**.
-    
-9. To add a statement of the `Respond` type so that the result of the above processing is returned as a response, click the last **+** icon in your low-code diagram.
+    6. Click **Save**.
 
-    ![Add Respond Statement](../assets/img/services/add-respond-statement.png)
-    
-    In the **Respond Expression** field, enter `response`, and then click **Save**.
+9. Add a Respond statement as follows so that the result of the above processing is returned as a response:
```

Comment:
why do we add a colon here? because we are introducing a list?

Reply:
Yes.

Reply:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108 
The plan is to go with the recommendation in https://developers.google.com/style/procedures#introductory-sentences with regard to the use of a colon when a sentence immediately precedes the procedure.